dms/models/directory.py

```diff
@@ -48,30 +48,24 @@ class DmsDirectory(models.Model):
         parent inherits the settings form its parent.""",
     )
 
-    root_storage_id = fields.Many2one(
-        comodel_name="dms.storage",
-        string="Root Storage",
-        ondelete="restrict",
-        compute="_compute_directory_type",
-        store=True,
-        readonly=False,
-        copy=True,
-    )
     # Override acording to defined in AbstractDmsMixin
     storage_id = fields.Many2one(
         compute="_compute_storage",
+        compute_sudo=True,
+        readonly=False,
         comodel_name="dms.storage",
         string="Storage",
         ondelete="restrict",
-        auto_join=True,
         store=True,
+        copy=True,
+        default=False,
     )
 
     parent_id = fields.Many2one(
         comodel_name="dms.directory",
         domain="[('permission_create', '=', True)]",
         string="Parent Directory",
-        ondelete="restrict",
+        ondelete="cascade",
```

Comment:
Why changing this? It's very risky to allow by default to unlink recursively. I think that should be an option under Actions, with double confirmation.

Reply:
Ok, I don't change it.